.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    # Require one of the following patterns to match the tag
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+.[0-9]+' # ex. v20.00.00.00
+      - 'v[0-9]+.[0-9]+.RC.[0-9]+'     # ex. v20.00.RC.00
+
+name: Create Release
+
+env:
+  PROJECT_NAME: FreeRTOS-metal
+  # Release is a prerelease if the tag contains rc
+  PRERELEASE: ${{ contains(github.ref, 'RC') }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

Comment:
Are we sure we want to run on the latest and not on one specific version that we choose to update time to time?